step.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# 'read' has to be before 'set -e'
+read -r -d '' UNAVAILABLE_MESSAGE << EOF_MSG
+Bitrise Build Cache is not activated in this build.
+
+You have added the **Activate Bitrise Build Cache for Bazel** add-on step to your workflow.
```

Comment:
Why is it an add-on?

Reply:
Not sure, we have the same one in the gradle step https://github.com/bitrise-steplib/bitrise-step-activate-gradle-remote-cache/blob/1f568f218c135875d221a9eec58dcac89c7de994/step.sh#L7.
Fixed it tough.

Reply:
I'll add this to our unrelated ticket about the remote cache to update it there too with the same release